nncf/onnx/hardware/fused_patterns.py

```diff
@@ -11,63 +11,518 @@
+    arithmetic_swish_sigmoid.join_patterns(barch_norm)
+    return arithmetic_swish_sigmoid
+
+
+@ONNX_HW_FUSED_PATTERNS.register(PatternsManager.ARITHMETIC_HARD_SIGMOID_SCALE_SHIFT)
+def create_arithmetic_hard_sigmoid_scale_shift():
+    arithmetic_swish_sigmoid = create_arithmetic_swish_sigmoid()
+    scale_shift = create_scale_shift_add()
+    arithmetic_swish_sigmoid.join_patterns(scale_shift)
+    return arithmetic_swish_sigmoid
+
+
+@ONNX_HW_FUSED_PATTERNS.register(PatternsManager.ARITHMETIC_SWISH_HARD_SIGMOID_BATCH_NORM)
+def create_arithmetic_swish_hard_sigmoid_batch_norm():
+    arithmetic_swish_hard_sigmoid = create_arithmetic_swish_hard_sigmoid()
+    barch_norm = batch_normalization_operations()
+    arithmetic_swish_hard_sigmoid.join_patterns(barch_norm)
+    return arithmetic_swish_hard_sigmoid
+
+
+@ONNX_HW_FUSED_PATTERNS.register(PatternsManager.ARITHMETIC_HARD_HARD_SIGMOID_SCALE_SHIFT)
+def create_arithmetic_hard_hard_sigmoid_scale_shift():
+    arithmetic_swish_hard_sigmoid = create_arithmetic_swish_hard_sigmoid()
+    scale_shift = create_scale_shift_add()
+    arithmetic_swish_hard_sigmoid.join_patterns(scale_shift)
+    return arithmetic_swish_hard_sigmoid
+
+
+@ONNX_HW_FUSED_PATTERNS.register(PatternsManager.BATCH_NORM_ACTIVATIONS)
+def create_batch_norm_activations():
+    batch_norm = batch_normalization_operations()
+    activations = atomic_activations_operations()
+    batch_norm.join_patterns(activations)
+    return batch_norm
+
+
+@ONNX_HW_FUSED_PATTERNS.register(PatternsManager.BATCH_NORM_SWISH_SIGMOID)
+def create_batch_norm_swish_sigmoid():
+    batch_norm = batch_normalization_operations()
+    swish_sigmoid = create_swish_with_sigmoid()
+    batch_norm.join_patterns(swish_sigmoid)
+    return batch_norm
+
+
+@ONNX_HW_FUSED_PATTERNS.register(PatternsManager.BATCH_NORM_SWISH_HARD_SIGMOID)
+def create_batch_norm_swish_hard_sigmoid():
+    batch_norm = batch_normalization_operations()
+    swish_hard_sigmoid = create_swish_with_hard_sigmoid()
+    batch_norm.join_patterns(swish_hard_sigmoid)
+    return batch_norm
+
+
+@ONNX_HW_FUSED_PATTERNS.register(PatternsManager.SCALE_SHIFT_ACTIVATIONS)
+def create_scale_shift_activations():
+    scale_shift = create_scale_shift_add()
+    activations = atomic_activations_operations()
+    scale_shift.join_patterns(activations)
+    return scale_shift
+
+
+@ONNX_HW_FUSED_PATTERNS.register(PatternsManager.SCALE_SHIFT_SWISH_SIGMOID)
+def create_scale_shift_swish_sigmoid():
+    scale_shift = create_scale_shift_add()
+    swish_sigmoid = create_swish_with_sigmoid()
+    scale_shift.join_patterns(swish_sigmoid)
+    return scale_shift
+
+
+@ONNX_HW_FUSED_PATTERNS.register(PatternsManager.SCALE_SHIFT_SWISH_HARD_SIGMOID)
+def create_scale_shift_swish_hard_sigmoid():
+    scale_shift = create_scale_shift_add()
+    swish_hard_sigmoid = create_swish_with_hard_sigmoid()
+    scale_shift.join_patterns(swish_hard_sigmoid)
+    return scale_shift
+
```

Comment:
General comment: you have many patterns for swish with sigmoid or swish with hardsigmoid which can be replaced by the patterns with activations, in which swish will be placed, as it was originally.
It will save you many lines of code

Reply:
Will it save the same transparency in patterns and the ability to quickly edit them?

Reply:
I believe - yes.
If I want to add a composite activation and want that this activation can be used in all patterns with activations - I just need to add the new activation composite pattern to `activations`. So, all patterns which are included `activations` will be updated with the recently added one.
So, I don't really need to care about updating every single pattern with Linear, BN, arithmetic and other ops combined with new composite activation

Reply:
What will you do when you need to remove 5 activations from 20 templates that are combined with other 10 under one variable?

Reply:
I will split the group - `activations` into two. The first will be used for these 20 templates. This is the group without 5 templates.
The second is the group with all activations which is used for others termplates, which needs all activations

Reply:
And it will increase the list of patterns 2x time.

Reply:
Only for these 20 templates we will change the `activations` pattern to the pattern without 5 activations. Other patterns will be without changes. So the number of lines increase by the numbers of line to create a new group of activations,.